src/cloud-api-adaptor/test/e2e/assessment_runner.go

```diff
@@ -135,6 +136,11 @@ func (tc *TestCase) WithInstanceTypes(testInstanceTypes InstanceValidatorFunctio
 	return tc
 }
 
+func (tc *TestCase) WithMultiplePodVM(alternateImageName string) *TestCase {
```

Comment:
I wonder if `WithMultiplePodVM` is the best name here? Maybe something like `WithAlternateImage` would be clearer?